compiler/src/main/java/io/neow3j/compiler/converters/ObjectsConverter.java

```diff
@@ -247,9 +315,9 @@ private static AbstractInsnNode convertConstructorCall(TypeInsnNode typeInsn,
     }
 
     // Checks if the given instruction is a call to the given classes constructor (i.e., <init>).
-    private static boolean isCallToCtor(AbstractInsnNode insn, ClassNode owner) {
+    private static boolean isCallToCtor(AbstractInsnNode insn, String ownerInternalName) {
```

Comment:
This change was necessary just to make the comparison in an easier way, right?

Or are there more reasons? Just trying to understand.

Reply:
Yes, just to make the method usable in more places. Only the name of the owner is needed inside of the method anyway, so why pass the whole ClassNode.